host.yaml

```diff
@@ -89,6 +85,7 @@ packages:
  - sudo coreutils less tar xz gzip bzip2 rsync tmux
  - nmap-ncat net-tools bind-utils strace
  - bash-completion
+ - hostname
```

Comment:
is there any particular reason we needed to add this? Is it pulled in via dependencies in RHCOS?

Reply:
With F28 repo, we don't need to specify hostname explicitly because it hostname gets pulled in as a dependency from some other package(don't know exactly which one) . But with F29 repo, hostname doesn't get pulled in as dependency and this lead to following error during rpm-ostree compose process
```
dracut: dracut module 'iscsi' will not be installed, because command 'hostname' could not be found!
dracut: dracut module 'nbd' will not be installed, because command 'nbd-client' could not be found!
dracut: dracut module 'iscsi' cannot be found or installed.
error: Finalizing rootfs: During kernel processing: Executing bwrap(rpmostree-dracut-wrapper): Child process killed by signal 1
```

Reply:
:+1: